### DISCUSSION TITLE
How can I get the deal price of a transaction?

### DISCUSSION THREAD

**Discussion by 0xUniko** [ProjectOpenSea/seaport#553] [Q&A]
If I buy an nft and the transaction is recorded on chain. How can I get the deal price of the transaction from the on-chain data?

**Comment by Slokh** [comment]
There's no real concept of "price" with Seaport as you can also swap NFT <> NFT, so it's slightly more tricky. 

You're interested in the `OrderFulfilled` event for this. https://github.com/ProjectOpenSea/seaport/blob/main/contracts/interfaces/ConsiderationEventsAndErrors.sol#L27-L34

You'll need to determine the deal price based on the `offer` and `consideration` depending on the use case, where `itemType` is `NATIVE` or `ERC20` for the traditional use cases of ERC721<>ERC20/ETH and ERC1155<>ERC20/ETH
